user_docs/en/userGuide.md

```diff
@@ -2169,7 +2171,7 @@ If "Read by paragraph" is checked, the selected start marker will be displayed t
 This can be especially helpful in applications used to read large pieces of text, like structured documents or books.
 In such documents, knowing where paragraphs start may be useful to understand the structure of the content, or to set bookmarks or annotations based on paragraph position.
 
-The options include using double space for softer paragraph breaks, or alternatively the paragraph symbol, Pilcrow (¶), for more obvious paragraph breaks.
+The options include using double space for softer paragraph breaks, or alternatively the paragraph symbol, pilcrow (¶), for more obvious paragraph breaks.
```

Comment:
I agree with @XLTechie's proposal. But in addition:
1. I'd not separate the two possibilities by a ";" (semicolon). The two options should remaine in the same phrase IMO. At least that would be more grammatically correct in French (my native language).
2. Also, I'd use "and" rather than "or" since both possibilities are in the options list, not one or the other.

As usual, for English fixes from myself, feel free to accept or decline.

```suggestion
The options include using double spaces, for softer paragraph breaks, and the paragraph symbol, pilcrow (¶), for more obvious paragraph breaks.
```

Reply:
Agree with dropping "alternatively", and changing "double space" to "double-spaces". I also agree with switching "or" to "and" here, as it's one or the other.
```suggestion
The options include using double-spaces for softer paragraph breaks, and the paragraph symbol, pilcrow (¶), for more obvious paragraph breaks.
```